__tests__/app.test.js

```diff
@@ -28,22 +28,59 @@ describe("GET /api", () => {
   });
 });
 describe("GET /api/topics", () => {
-  test("200: Responds with an object of all the topics", () => {
+  test("200: Responds with an array of all the topics including slug and description properties", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then(({body}) => {
-        expect(body.result).toEqual(data.topicData)
+        expect(body.topics).toEqual(data.topicData)
       })
   })
 })
+describe("GET /api/articles/:article_id", () =>{
+  test("200: Responds with the correct article object with all the correct properties", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+      const article = body.article
+      expect(article.title).toBe("Living in the shadow of a great man")
+      expect(article.topic).toBe("mitch")
+      expect(article.author).toBe("butter_bridge")
+      expect(article.body).toBe("I find this existence challenging")
+      expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+      expect(article.votes).toBe(100)
+      expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+     })
+  })
+  test("400: Responds with an error message when the article id is invalid", () =>{
+    return request(app)
+    .get("/api/articles/banana")
+    .expect(400)
+    .then(({body}) => {
+      const error = body
+      expect(error.msg).toEqual("Invalid article id")
```

Comment:
I think the message here is a good idea, but you may soon find out that using 'bad request' and 'not found' are easier to deal with because they encompass more use cases. If you were to go down this path, you'll end up having to create more error handling situations. Plus these messages are for front end devs, not users, therefore they can figure out themselves what the issue is and display whatever they'd like on the front end.